kubernetes/structures.go

```diff
@@ -402,9 +402,9 @@ func flattenResourceQuotaScopeSelectorMatchExpressions(in []api.ScopedResourceSe
 	if len(in) == 0 {
 		return []interface{}{}
 	}
-	out := make([]interface{}, 1)
+	out := make([]interface{}, 0)
```

Comment:
in this case, I think we actually know the length of the `out` slice. It will be equal to the length of `in`.
```suggestion
	out := make([]interface{}, len(in))
```
This should be the only change needed and we can keep the for loop below as it was, using indexes.

Reply:
Yes, you are right. Thank you, @alexsomesan!